eZ/Publish/Core/Search/Legacy/Content/WordIndexer/Gateway/DoctrineDatabase.php

```diff
@@ -110,12 +110,7 @@ public function index(FullTextData $fullTextData)
             } else {
                 $integerValue = 0;
             }
-            $text = $this->transformationProcessor->transform(
-                $fullTextValue->value,
-                !empty($fullTextValue->transformationRules)
-                    ? $fullTextValue->transformationRules
-                    : $this->fullTextSearchConfiguration['commands']
-            );
+            $text = $this->transformationProcessor->transformByGroup($fullTextValue->value, 'lowercase');
```

Comment:
`$fullTextValue->transformationRules` is an extension point to add some custom logic for transforming values

Reply:
That is indeed true. I think what you want done here is to apply both transformations - the one we require (lowercase), followed by any custom transformations from `transformationRules`. Correct?